client/state/test/initial-state.js

```diff
@@ -10,17 +11,26 @@ import sinon from 'sinon';
 import config from 'config';
 
 describe( 'initial-state', () => {
-	let localforage, createReduxStoreFromPersistedInitialState, MAX_AGE;
+	let localforage, createReduxStoreFromPersistedInitialState, MAX_AGE,
+		isSwitchedUser = false,
+		isSupportUserSession = () => isSwitchedUser;
 
 	require( 'lib/react-test-env-setup' ).auto();
 
 	before( () => {
+		mockery.registerMock( 'lib/user/support-user-interop', { isSupportUserSession: isSupportUserSession } );
+		mockery.enable( { warnOnUnregistered: false } );
```

Comment:
Hmm, looks like my sinon stubs for config don't work if I use the `use-mockery` helper. I'll investigate later, but I'm going to use the cache clean option for now.